usrp3/tools/scripts/uhd_image_builder_gui.py

```diff
@@ -542,6 +538,27 @@ def grc_populate_list(self, parent, files):
             self.show_not_xml_warning()
             return
 
+    def refresh_oot_dirs(self):
+        """
+        Populates the OOT directory list from the OOT include file
+        """
+        oot_include = os.path.join(uhd_image_builder.get_scriptpath(), '..', '..', 'top',\
+            self.target, 'Makefile.OOT.inc')
+        dir_list = []
+        with open(oot_include, 'r') as fil:
+            text = fil.readlines()
+            for lines in text:
+                lines = lines.partition('$(BASE_DIR)/')
+                if (lines[1] == '$(BASE_DIR)/'):
+                    relpath = lines[2].replace('\n', '')
+                    ootpath = os.path.abspath(os.path.join(uhd_image_builder.get_scriptpath(), '..', '..', 'top', relpath))
+                    dir_list.append(ootpath)
+        if (len(dir_list) == 0):
+            self.oot.removeRows(0, parent.rowCount())
```

Comment:
There is no "parent" defined here, so this raises "NameError". Maybe you meant here "self.oot.rowCount()"?